src/components/OpenSeadragonViewer.js

```diff
@@ -58,6 +58,14 @@ export class OpenSeadragonViewer extends Component {
 
     this.setState({ viewer });
 
+    viewer.addHandler('canvas-press', () => {
```

Comment:
would `canvas-drag` and `canvas-drag-end` be more appropriate events?

Reply:
@cbeer: Seems to work the same for me in my browsers. Is that better for touch screen devices? If so, I can commit the change.

Reply:
This commit changes it to use `canvas-drag` and `canvas-drag-end`:
https://github.com/ProjectMirador/mirador/pull/3644/commits/fd2681febf0fb3d6c01148cc2a271b740b4a6aa4